### DISCUSSION TITLE
Angular support

### DISCUSSION THREAD

**Discussion by FranCarrascosa** [cawa-93/vite-electron-builder#891] [Q&A]
Hi!

Reading the description of this repo it seems that supports Angular framework but I don't find anything.

> Secure boilerplate for Electron app based on Vite. TypeScript + Vue/React/Angular/Svelte/Vanilla

Does anyone have a sample supporting it?

**Comment by cawa-93** [comment]
I don't remember anyone reporting about their fork for Angular.

This template was designed as several individual parts. One of them is UI. Placed in `packages/renderer`. From docs:
> [packages/renderer](https://github.com/cawa-93/vite-electron-builder/blob/main/packages/renderer). Responsible for the contents of the application window. In fact, it is a regular web application. In developer mode, you can even open it in a browser. The development and build process is the same as for classic web applications. Access to low-level API electrons or Node.js is done through the preload layer.

Technically, you can render any web application on any framework or technology in same way as you usually do for regular web apps. The building should also be similar to how you normally build. For example this template configured to use Vue, and [using Vite to building](https://github.com/cawa-93/vite-electron-builder/blob/09be29ace59029b22fb37442530fe45622ce34ab/package.json#L16). Sadly, but [Vite doesn't provide Angular support](https://github.com/vitejs/vite/issues/543#issuecomment-658473854). But you also may use any build tool what you want. Just change `build:renderer` script. It is only important that, bundled web application is placed in `packages/renderer/dist/`

Also you may need to change `scripts/watch.js`. By default there starts 3 vite watchers to handle files changes and partically update app.

Angulat is the edge case. I believe, is technically possible, but requires a more complex configuration. However, feel free to report if you find any evidence that Angular cannot be used in this template. Then I will remove all mention of him in the description.
